theano/__init__.py

```diff
@@ -31,10 +31,6 @@
 
 import sys
 
-if sys.platform == 'win32' and sys.version_info[0:2] == (3, 5):
-    raise RuntimeError(
-        "Theano do not support Python 3.5 on Windows. Use Python 2.7 or 3.4.")
```

Comment:
All conda Pythons are built with MSVC, and I can confirm that there are no problems here, also none when using mingw-w64 GCC.